### PR TITLE
Implementation

### PR DESCRIPTION
**Pull request by ChrispaixK**
# OOP school library App 📚 
> Add Person, Student, and Teacher classes
> The Student and Teacher class inherits from  Person class

In this PR I :
* Setup working environment
* Edit Readme and Licence files
* Create class `Person`
* Create class `Student`
* Create class `Teacher`

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require_relative './person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, classroom)
+    super(age)
+    @classroom = classroom
+  end
```
Comment (mukhammadrizooff) [inline student.rb]: - [ ] [REQUIERED] Good job so far! i think you missed `name` and `parent_permission`, your `def initialize` should be like that:

```ruby
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require_relative './person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(age, specialization)
+    super(age)
+    @specialization = specialization
+  end
```
Comment (mukhammadrizooff) [inline teacher.rb]: - [ ] [REQUIRED] Good job so far! I think you missed `name` and `parent_permission`, your `def initialize` should be like that:

```ruby
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @ChrispaixK, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @ChrispaixK👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**